simple_clock/app.js

```diff
@@ -1,3 +1,92 @@
-window.addEventListener("load", function() {
-  console.log("Hello World!");
+var timetext;
+var watchtimetext;
+var start;
+var watchtime;
+var elapsed;
+var timoutid;
+var spbutton;
+var rebutton;
+var opbutton;
+var minute;
+var second;
```

Comment:
Bunch of global variables.

You could encapsulate them (and all other functions) into an anonymous function to avoid naming conflicts.